specifications/rsp/rsp.yaml

```diff
@@ -318,14 +333,8 @@ components:
                   - "03"
               settlement_reference_no:
                 type: string
-                description: "Reference number, in case of successful settlement"
+                description: "Reference number, incase of successful settlement"
                 example: "3718683618631"
-              prev_settlement_reference_no:

```

Comment:
Previous reference numbers are required but not mandatory attributes. Please add them in your file as well.